tests/baselines/reference/enumBasics.js

```diff
@@ -145,7 +145,7 @@ var E8;
 var E9;
 (function (E9) {
     E9[E9["A"] = 0] = "A";
-    E9[E9["B"] = 0] = "B";
+    E9["B"] = E9.A;

```

Comment:
> That's what I would expect, this pattern works well with string enums

But string enums weren't officially approved at all.

> Don't think the current behavior is more correct where the last declared member overrides the mapping

Perhaps for most cases, but this is still a breaking change; the compiler did not give an error in these scenarios.

Can you explain why this change was originally necessary?